packages/components/HowItWorksSection.tsx

```diff
@@ -12,53 +12,52 @@ const HomeSection = () => {
         textAlign="center"
         margin={20}
       >
-        <span>
+        <Text as="span">
           <Icon
             as={GoCommentDiscussion}
             w={7}
             h={7}
             marginBottom={3}
             color="purple.500"
           />
-
-          <h3 style={{ fontWeight: 'bold', marginBottom: '0.5rem' }}>
+          <Heading size="sm" marginBottom="0.5rem">
             Participe da comunidade
-          </h3>
-          <p>
+          </Heading>
+          <Text>
             Parcicipe de meetups semanais com diversos outros profissionais da
             área de tecnologia e expanda seus conhecimentos.
-          </p>
-        </span>
-        <span>
+          </Text>
+        </Text>
+        <Text as="span">
           <Icon as={GoCode} w={7} h={7} marginBottom={3} color="purple.500" />
-          <h3 style={{ fontWeight: 'bold', marginBottom: '0.5rem' }}>
+          <Heading size="sm" marginBottom="0.5rem">
             Aprenda a programar
-          </h3>
-          <p>
+          </Heading>
+          <Text>
             Aprenda a programar de uma forma dinâmica através do
             compartilhamento de conhecimento e participe de nossas mentorias
             pesonalizadas ou em grupo.
-          </p>
-        </span>
-        <span>
+          </Text>
+        </Text>
+        <Text as="span">
           <Icon
```

Comment:
Note que a estrutura Icon + Heading + Text se repete em 3 lugares. O que acha de transformar isso num componente? Deva forma vc melhora o reaproveitamento de codigo e para de se repetir

Reply:
Não precisa criar um novo arquivo para isso, pode fazer esse componente nesse aquivo aqui, só não precisa exportar ele, pois só será usado aqui dentro

Reply:
Minha regra geral para quebrar algo em componentes é: 
- essa estrutura é usada mais de uma vez?
- a estrutura tem muita logica específica?

Se a resposta for sim para alguma das duas questões, vale criar um novo componente

Se ele for compartilhado entre varios componentes, ai vale criar ele em um arquivo separado

Reply:
a ideia sempre foi caminhar para estruturar o componente desta forma. so nao foi feito no inicio mesmo pois ainda nao dava pra saber o que seria usado. acho que agora ja eh uma boa hora pra montar ele assim mesmo